1-src/2-services/10-utilities/mock-utilities/mock-generate.mts

```diff
@@ -0,0 +1,292 @@
+            circle.memberList.push(member);
+        }
+    }
+
+    await DB_DELETE_CONTACT_CACHE_BATCH(circle.memberList.map(profile => profile.userID));
+
+    log.event(`Mock Circle: ${circle.circleID} | ${circle.name} | created for leader ${leaderID} with ${circle.memberList.length} members.`);
+    return circle;
+}
+
+
+/******************************************
+* Auto Populate Profile for Demo Purposes *
+*******************************************/
+export const populateDemoRelations = async(user:USER):Promise<USER> => {
+    if(user.userID <= 0 || !user.isValid) {
+        log.warn('Error populating profile relations with invalid USER', user.userID, user.toString());
+        return user;
+    }
+
+    const modifiedUserIDList:number[] = [user.userID];
+
+    /* Add Partnerships */
+    if(user.walkLevel === undefined) user.walkLevel = 5;  //DB default; not assigned yet on /signup
+    const availablePartnerList:NewPartnerListItem[] = await DB_SELECT_AVAILABLE_PARTNER_LIST(user, 4);
+    for(let i=0; i<availablePartnerList.length; i++) {
+        modifiedUserIDList.push(availablePartnerList[i].userID);
+        const userID:number = getUserID(user.userID, availablePartnerList[i].userID);
+        const partnerID:number = getPartnerID(user.userID, availablePartnerList[i].userID);
+
+        await DB_ASSIGN_PARTNER_STATUS(userID, partnerID, ((i % 2) === 0) ? DATABASE_PARTNER_STATUS_ENUM.PARTNER 
+            : (userID === user.userID) ? DATABASE_PARTNER_STATUS_ENUM.PENDING_CONTRACT_USER : DATABASE_PARTNER_STATUS_ENUM.PENDING_CONTRACT_PARTNER);
+    }
+
+    for(let i=availablePartnerList.length; i<4; i++) {
+        const partner:USER = await createMockUser(false);
+        const modifiedPartner:USER = USER.constructByClone(partner);
+        modifiedPartner.walkLevel = user.walkLevel;
+        modifiedPartner.gender = user.gender;
+        modifiedPartner.dateOfBirth = user.dateOfBirth;
+        await DB_UPDATE_USER(partner.userID, modifiedPartner.getUniqueDatabaseProperties(partner));
+        await DB_ASSIGN_PARTNER_STATUS(user.userID, partner.userID, ((i % 2) === 0) ? DATABASE_PARTNER_STATUS_ENUM.PARTNER : DATABASE_PARTNER_STATUS_ENUM.PENDING_CONTRACT_USER);
+    }
+
+    /* Join Circles */
+    const circleList = await getMockCircleList(4, 11);
+    for(let i=0; i<circleList.length; i++) {
+        await DB_INSERT_CIRCLE_USER_STATUS({circleID: circleList[i].circleID, userID:user.userID, status: ((i % 2) === 0) ? DATABASE_CIRCLE_STATUS_ENUM.MEMBER : DATABASE_CIRCLE_STATUS_ENUM.INVITE});
+    }
+
+    /* Share Prayer Requests */
+    for(const request of await getMockPrayerRequestList(3, 50)) { //Likely already shared through circles
+        await DB_INSERT_USER_RECIPIENT_PRAYER_REQUEST({prayerRequestID: request.prayerRequestID, userID: user.userID})
+    }
+
+    /* Add Owned Prayer Request */
```

Comment:
Brackets aren't necessarily required here but can be nice if we add to the loop later

Reply:
You're right, suppose loops I've always done brackets since school.  Now at work, they strictly require brackets even for simple if statements; boss's preference.  When I switch projects, I find myself going back to delete brackets on the simple if's.